wallet/wallet-cli-lib/src/commands/command_handler/mod.rs

```diff
@@ -375,6 +376,59 @@ where
                 Ok(ConsoleCommand::Print(addresses_table.to_string()))
             }
 
+            ColdWalletCommand::ShowStandaloneAddresses => {
+                let (wallet, selected_account) = wallet_and_selected_acc(&mut self.wallet).await?;
+                let addresses_with_label =
+                    wallet.get_standalone_addresses(selected_account).await?;
+
+                let addresses_table = {
+                    let mut addresses_table = prettytable::Table::new();
+                    addresses_table.set_titles(prettytable::row!["Address", "Label",]);
```

Comment:
I think the type should be included too, like whether it's multisig or something else, but I'm not sure how to do that.